libs/clients/middlewares/src/lib/createEnhancedFetch.spec.ts

```diff
@@ -209,7 +209,7 @@ describe('EnhancedFetch', () => {
 
   it('can be configured to open circuit for 400 errors', async () => {
     // Arrange
-    env = setupTestEnv({ treat400ResponsesAsErrors: true })
```

Comment:
This test case needs to be removed completely as this will no longer be valid when we remove this feature

Reply:
Removed :heavy_check_mark: